helloworld/src/main/scala/com/salesforce/hw/titanic/OpTitanic.scala

```diff
@@ -86,6 +86,7 @@ object OpTitanic extends OpAppWithRunner with TitanicFeatures {
     .setLabelCol(survived)
     .setPredictionCol(pred)
     .setRawPredictionCol(raw)
+    .setProbabilityCol(prob)
```

Comment:
ooo - is this what we have in master?? 'cause this is NOT up to date with the Evaluators version currently in master

Reply:
master is working against TAI 0.3.4. I will update 0.4.0 release branch as well.